etc/settings.yaml.in

```diff
@@ -66,4 +66,10 @@ tools:
     args: ""
   vep:
     executable: @VEP@
+    args: ""
+  gunzip:
+    executable: @GUNZIP@
+    args: "--keep"
```

Comment:
Remember that users can overwrite this.  If you really need `--keep` for that rulel I'd remove it from here.  The `args` setting is useful for "unusual" arguments.  `--keep` on the other hand has the ability to disrupt the pipeline when removed.

Not very important, just a general comment to consider what options to add to the settings file.  There's no hard rule here.